BLOCKFILE

```diff
@@ -21,69 +21,69 @@ BLOCKFILE
 ci/scripts/check-pr-changes-allowed.py
 
 # Earlgrey related RTL
-hw/ip/*/rtl/*
-hw/ip_templates/*/rtl/*
-hw/top_earlgrey/ip/*/rtl/*
-hw/top_earlgrey/ip_autogen/*/rtl/*
+## hw/ip/*/rtl/*
```

Comment:
Why the double '##'?

Reply:
oh that is just a hack so that it is easy to search and block-remove them, without touching the inline comments.